packages/components/src/toolbar/toolbar/types.ts

```diff
@@ -12,4 +12,14 @@ export type ToolbarProps = {
 	 * An accessible label for the toolbar.
 	 */
 	label: string;
+	/**
+	 * Specifies the toolbar's style.
+	 *
+	 * Leave undefined for the default style. Or 'unstyled' which
+	 * removes the border from the toolbar, but keeps the default
+	 * popover style.
+	 *
+	 * @default undefined
+	 */
+	variant?: 'unstyled' | undefined;
```

Comment:
I called this 'unstyled' to match what is used for a borderless style in the `Popover` component. 

@ciampo - you've been mentioning 'borderless' so if you prefer another name, I'm happy to change. I just thought this worked for consistency.

Reply:
Along these lines, I also wondered if we'd want `Toolbar` to have a `'popover'` `variant` to match `Popover`'s `undefined` style. But I don't think it would necessarily be that helpful; for example, in the one override addressed in this PR, the entire border is removed, and then just a bottom border is added to replace it: 

https://github.com/WordPress/gutenberg/blob/c6718435c545175675d883d1f5003b5f97a53b8e/packages/block-editor/src/components/block-tools/style.scss#L112-L113